templates/repo/projects/view.tmpl

```diff
@@ -103,6 +103,11 @@
 										{{svg "octicon-pin"}}
 										{{$.locale.Tr "repo.projects.column.set_default"}}
 									</a>
+								{{else}}
+									<a class="item show-modal button" data-modal="#unset-default-project-board-modal-{{.ID}}">
```

Comment:
IIRC there is a general method to pass variables to a model dialog, then there is no need to create a lot of model elements on the page?

Reply:
Hmm, not blocker, if this is ready for merge, I will do following improvements in  next PR

Reply:
I’m trying to improve the code here.

Reply:
I’m trying to improve the code here.

Reply:
emmm, no idea about converting the commit button's class:
- use `red` when it is `set default` button
- use `primary` when it is `unset default` button

Reply:
Thank you. Never mind, IMO no need to block as long as it works. The "project/board" pages have too much technical debt in history and might need some more improvements.

Reply:
Let's move on , maybe some maintainers could help to merge 😁